classes/classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << students
```

Comment:
- [ ] Kindly make sure to fix the typo on `add_student` method, you should push `student` to the array not `students`
```suggestion
    @students << student
```